src/app/containers/IndexHeading/index.stories.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { cyrillicAndLatin } from '@bbc/gel-foundations/scripts';
+import IndexHeadingContainer from '.';
+
+storiesOf('Containers|Index Heading', module)
```

Comment:
If I understand correctly`IndexHeading` will be used not only for IDX but also for Most Read page? In that case should we have knobs for this story to see what other services look like?

Reply:
Good idea 👍

Reply:
I've added the `ServiceContextProvider` to the story so just need to pass the `service`.